packages/drivers/driver-base/api-report/driver-base.api.md

```diff
@@ -26,6 +26,8 @@ export class DocumentDeltaConnection extends EventEmitterWithErrorHandling<IDocu
     protected constructor(socket: Socket, documentId: string, logger: ITelemetryLoggerExt, enableLongPollingDowngrades?: boolean, connectionId?: string | undefined);
     // (undocumented)
     protected addTrackedListener(event: string, listener: (...args: any[]) => void): void;
+    // (undocumented)
```

Comment:
Please be sure to add documentation to any new API members.

Reply:
Decided to do this work in a separate PR.